XCUITests/NavigationTest.swift

```diff
@@ -293,4 +293,83 @@ class NavigationTest: BaseTestCase {
         XCTAssertTrue (desktopViewElement.exists, "Desktop view is not available after coming from another site in mobile view")
      }
      */
+    func testLongPressLinkOptions() {
+        navigator.openURL(website_2["url"]!)
+        app.webViews.links["More information..."].press(forDuration: 2)
+        waitforExistence(app.sheets["http://www.iana.org/domains/example"])
```

Comment:
can we put the URL as a constant and reuse?

Reply:
Yeah, sorry I should have done that before...will change it!